src/pages/dashboard/dashboardComponenets/mainCard.tsx

```diff
@@ -20,7 +36,7 @@ const MainCard: React.FC = () => {
 						<div className="card border-dark">
 							<div className="card-body text-center">
 								<h6 className="card-title">Inquiries</h6>
-								<p className="card-text">45</p>
+								<p className="card-text">{stateInquiry.inquiries.length}</p>
```

Comment:
Please try to add ternary condition to this line because if there is an error in when reading length, this will cause an error. Check whether 'stateApplication.application' is available or else simply display 0.